src/accelerate/accelerator.py

```diff
@@ -1952,6 +1955,11 @@ def prepare_optimizer(self, optimizer: torch.optim.Optimizer, device_placement=N
         >>> optimizer = accelerator.prepare_optimizer(optimizer, device_placement=True)
         ```
         """
+        if is_lomo_available():
+            from lomo_optim import AdaLomo, Lomo
```

Comment:
This should be done at the top of the file

Reply:
Doing that leads to an error due to circular imports 😢 this is because `lomo_optim` imports stuff from transformers, that itself import stuff from accelerate. Added a comment there